FE/todo-app/src/components/TopNavigation/TopNavigation.tsx

```diff
@@ -1,20 +1,33 @@
-import { FC } from "react";
+import { FC, useState } from "react";
 import { BsPlus, BsCheckLg, BsCollection } from "react-icons/bs";
 import styles from "./TopNavigation.module.css";
 import { Search } from "./TopNavigationSearch";
 import { Title } from "./TopNavigationTitle";
 import { TopNavigationIcon } from "./TopNavigationIcon";
 import { UserCircle } from "./TopNavigationUserCircle";
+import { CreateGroupInput } from "../createGroupForm/CreateGroupInput";
 
 export const TopNavigation: FC = () => {
+  const [open, setOpen] = useState(false);
+
+  const handleOpen = () => {
+    setOpen(true);
+  };
+
```

Comment:
na tohle udělej nějaký univerzální customHook, určtitě to bude potřebovat na více místech... mělo by to fungovat tak že se ti z něj vrátí 4 proměné... [isOpen, close, open, toggle]